perun-db/oracle.sql

```diff
@@ -866,7 +866,7 @@ create table tasks (
 	status nvarchar2(16) not null,
 	start_time date,
 	end_time date,
-	engine_id integer not null,
+	engine_id integer,
```

Comment:
Please update DB version in a comment on first line of a file and at the end in initial version update. Thank you.